addons/knobs/configure.js

```diff
@@ -0,0 +1,12 @@
+import {withKnobs as addonWithKnobs} from '@storybook/addon-knobs';
+
+function withKnobs(opts) {
+	return addonWithKnobs({
+		// debounce: {wait: 500}, // Same as lodash debounce.
```

Comment:
I know this PR simply calls for copying the enact sampler/storybook config, but I'm wondering if it's a good time to clean up some lingering commented-out code. I'm not sure the purpose of this (or if it's needed for any reason). Should this get cleaned up?

Reply:
I'd originally left it in case we wanted to restore it, but we haven't needed it yet, so it's fine to dispense with it.